src/openrct2/entity/Guest.cpp

```diff
@@ -2321,6 +2327,11 @@ void Guest::SpendMoney(money64& peep_expend_type, money64 amount, ExpenditureTyp
 
     WindowInvalidateByNumber(WindowClass::Peep, Id);
 
+#ifdef ENABLE_SCRIPTING
+    // guest pays original price, player receives modified amount
+    GetContext()->GetScriptEngine().RunMoneySpendHooks(amount, expenditure);
```

Comment:
wouldn't you want this to be -amount? I.e. *-1 before handing it to the hook.

Reply:
yes indeed